src/components/Errors.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Error } from '../types/Error';
+
+type Props = {
+  errorMessage: Error,
+  errorVisibility: boolean
+  onHidden: (visibility: boolean) => void,
+};
+
+export const Errors: React.FC<Props> = ({
+  errorMessage,
+  errorVisibility,
+  onHidden,
+}) => {
+  const onCloseClick = () => {
```

Comment:
```suggestion
  const handleCloseClick = () => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364